CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.6]
```

Comment:
Do not add a version number yet, we keep everything in the ## Unreleased section until we know where it will be released (could be a patch release if we decide to release a patch, or a minor, or a major, this is not known yet).

Reply:
Noted, I'll remove it.